react/features/base/responsive-ui/actions.js

```diff
@@ -12,9 +12,7 @@ import type { Dispatch } from 'redux';
  * very brittle because it's completely disconnected from the UI which wants to
  * be rendered and, naturally, it broke on iPad where even the secondary Toolbar
  * didn't fit in the height. We do need to measure the actual UI at runtime and
- * determine whether and how to render it. I'm bumping from 240 to 300 because I
- * don't have the time now to refactor {@code ReducedUIDetector} or rip it out
- * completely.
+ * determine whether and how to render it. I'm bumping from 240 to 300.
```

Comment:
Do we still need the "I'm bumping..." part?

Reply:
Nah, I can axe that.